lib/debounce.js

```diff
@@ -1,16 +1,11 @@
-import requestAnimationFrame from './requestAnimationFrame'
+export default function debounce (func, time = 300) {
+  let timeoutId = null
 
-export default function debounce (fn) {
-  let waiting = false
-
-  return function (...args) {
-    if (!waiting) {
-      waiting = true
-
-      requestAnimationFrame(() => {
-        fn(...args)
-        waiting = false
-      })
+  return function () {
```

Comment:
You were right. We managed to fix the issues by simply being more smart in the Collapsible implementation and preemptively abort calculating not needed sizes.